controllers/storagecluster/external_resources.go

```diff
@@ -380,6 +381,9 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				scc = newCephBlockPoolStorageClassConfiguration(instance)
 				// update the storageclass name to rados storagesclass name
 				scc.storageClass.Name = fmt.Sprintf("%s-%s", instance.Name, d.Name)
+			} else if d.Name == cephRbdTopologyStorageClassName && instance.Spec.ManagedResources.CephNonResilientPools.Enable {
+				scc = newNonResilientCephBlockPoolStorageClassConfiguration(instance)
+				scc.storageClass.Parameters["topologyConstrainedPools"] = getTopologyConstrainedPoolsExternalMode(d.Data)
```

Comment:
@parth-gr I think it is better to have this constructed and passed in the JSON blob itself instead of building it in the ocs-operator.

Reply:
@iamniting I would like to use the power of go code for this case as it is more straightforward, in the python script we have to create a template like we did it in import shell script, https://github.com/rook/rook/pull/13821/files#diff-17c14009bbe3863a3d47d44e3648b462b4269bb8a98f38492bc597321d053816 

if you have any easy maintainable way in python please let me know.

Reply:
moreover script is just to extract/scrape information, styling would be done in the next level of code.

Reply:
okay in that case we will do the magic here itself.